packages/block-editor/src/components/copy-handler/index.js

```diff
@@ -113,6 +113,7 @@ export function useClipboardHandler() {
 				return;
 			}
 
+			const eventDefaultPrevented = event.defaultPrevented;
 			event.preventDefault();
```

Comment:
Oh hah I missed that this was called here earlier, so checking for event.defaultPrevented should work. 👍